lib/icingadb/icingadb-objects.cpp

```diff
@@ -115,6 +115,17 @@ void IcingaDB::ConfigStaticInitialize()
 	});
 }
 
+/**
+ * Puts a no-op command with a result at the end of the queue and wait for the result,
+ * i.e. for everything enqueued to be processed by the server.
+ *
+ * @ingroup icingadb
+ */
+static void SyncRedisConnection(const RedisConnection::Ptr& rCon)
+{
+	rCon->GetResultOfQuery({"PING"}, (RedisConnection::QueryPriority)42);
```

Comment:
When we actually consider merging this, this should get a proper priority value.

Reply:
... or the just done refactoring.